curriculum/js2/3.js

```diff
@@ -5,10 +5,31 @@
  * @returns {array}
  */
 
-const solution = (a) => {
-  return [];
+const solution = (a, idx = 0) => {
+  if (idx === a.length) {
+    return a;
+  }
+
+  if (!isPrime(a[idx])) {
+    a[idx] = 1;
+  }
+
+  return solution(a, idx + 1);
+};
+
+const isPrime = (num, val = 2) => {
```

Comment:
instead of writing this over again, try importing it. `const {isPrime} = require('../js1/6.js');` Is it problem 6? I forgot.